pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -14,6 +15,12 @@ import (
 const (
 	// DefaultFlavour is used when no specialized configuration exists.
 	DefaultFlavour = "osd-4"
+
+	// MadeByOsdE2E property to attach to clusters.
+	MadeByOsdE2E = "MadeByOsdE2E"
```

Comment:
This is pure opinion but I like "OSDe2e" better.

Reply:
Dang, I was actually trying to match your preference but I inverted it in my head, hahaha. Will change.